test/test_helper.exs

```diff
@@ -19,3 +19,34 @@ defmodule ElixirBenchWeb.AbsintheHelpers do
   end
 end
 
+defmodule ElixirBenchWeb.TestHelpers do
+  @moduledoc false
+
+  import ExUnit.Assertions
+
+  def assert_required_fields_message(required_fields, resp_body) do
+    Enum.map(required_fields, fn field ->
+      assert(
+        required_field_message_in_string?(field, resp_body),
+        "Error message not found for field \"#{field}\" in #{resp_body}"
+      )
+    end)
+  end
+
+  def assert_empty_response_data(query, response) do
```

Comment:
Small thing again (sorry!) both for this function and the function above - it'd be better to have the `response` or `response_body` as the first argument to the function?

Why you ask? First - in elixir the main data structure should be the first argument because second - you can then use it conveniently in pipes.

Not saying we should use this function in pipes right now - but the main argument as the first arguments really helpes me

Reply:
the best reviews ever <3 Fixed! 

I am taking some notes from your tips. I will try to read them weekly to memorize everything.